keps/sig-cluster-lifecycle/kubeadm/0023-kubeadm-config.md

```diff
@@ -212,6 +214,18 @@ active (a setting already introduced in `v1beta1`) and an ability to ignore some
 Nevertheless, SIG Cluster Lifecycle should be careful not to add fields that are needed for fairly
 limited use cases.
 
+### Public types extraction
```

Comment:
I  think we should describe what impacts this change has on the users (change the import path only)

Reply:
Indeed, this needs a sentence or two.